### PR TITLE
build(docs-infra): replace base64 images for stackblitz by svgs

### LINKED ISSUE #48773: Docs: .png images are broken in StackBlitz live samples
### Describe the problem that you experienced

Live samples deployed on StackBlitz have broken `.png` images.

Steps to reproduce:
1. Open page https://angular.io/guide/property-binding
2. Open the live sample link on the page: https://angular.io/generated/live-examples/property-binding/stackblitz.html
3. Let StackBlitz install and run the code
4. In the output you'll see broken images:
aio_bug

I think following code for Plunker still remained in StackBlitz builder, which is causing the issue:
https://github.com/angular/angular/blob/9e1e5ccdfe1f25a9995f70e3c64ae72551af7562/aio/tools/stackblitz-builder/builder.mjs#L185-L187
The code converts and renames all `.png` files. I think all the live samples on StackBlitz show broken images.

### Enter the URL of the topic with the problem

https://angular.io/guide/property-binding

### Describe what you were looking for in the documentation

Live samples working out of the box.

### Describe the actions that led you to experience the problem

Run live samples containing `.png` images on StackBlitz.

### Describe what you want to experience that would fix the problem

Stop converting to base64. These demos need simple images.
The [offline downloadable code](https://angular.io/generated/zips/property-binding/property-binding.zip) uses original normal images.

### PR DESCRIPTION
**Pull request by JeanMeche**
As stackblitz [does't support binary files for dynamicaly created projects](https://developer.stackblitz.com/platform/api/post-api), we now use only svgs.

Fixes: https://github.com/angular/angular/issues/48773

## PR Type
What kind of change does this PR introduce?

- [x] Documentation content changes

### CODE REVIEW

**Comment by JeanMeche** [comment]
Can I get the `aio:preview ` tag to confirm everything works fine nox ?

**Comment by alan-agius4** [comment]
> Can I get the `aio:preview ` tag to confirm everything works fine thx ?

Done

File: aio/tools/stackblitz-builder/builder.mjs [hunk 1]
```diff
@@ -182,12 +182,7 @@ export class StackblitzBuilder {
     config.fileNames.forEach((fileName) => {
       let content;
       const extn = path.extname(fileName);
-      if (extn === '.png') {
-        content = this._encodeBase64(fileName);
-        fileName = `${fileName.slice(0, -extn.length)}.base64${extn}`;
-      } else {
-        content = fs.readFileSync(fileName, 'utf-8');
-      }
+      content = fs.readFileSync(fileName, 'utf-8');
```
Comment (alan-agius4) [inline aio/tools/stackblitz-builder/builder.mjs:185]: Nit: the `_encodeBase64` method should be removed as it's unused.

**Comment by alan-agius4** [comment]
Nit: commit message should be `build(docs-infra)`

File: aio/tools/stackblitz-builder/builder.mjs (hunk 1 above)
Comment (OnkarRuikar) [inline aio/tools/stackblitz-builder/builder.mjs:185]: Won't reading `.png`s using `utf-8` encoding create problem?
How about this?
```suggestion
      if (extn === '.png') {
        content = fs.readFileSync(fileName);
      } else {
        content = fs.readFileSync(fileName, 'utf-8');
      }
```

**Comment by JeanMeche** [comment]
For the reviewer, here are impacted examples: 
* https://ng-dev-previews-fw--pr-angular-angular-48774-ztfhlji9.web.app/generated/live-examples/property-binding/stackblitz.html
* https://ng-dev-previews-fw--pr-angular-angular-48774-ztfhlji9.web.app/generated/live-examples/ajs-quick-reference/stackblitz.html
* * https://ng-dev-previews-fw--pr-angular-angular-48774-ztfhlji9.web.app/generated/live-examples/interpolation/stackblitz.html
* * https://ng-dev-previews-fw--pr-angular-angular-48774-ztfhlji9.web.app/generated/live-examples/template-syntax/stackblitz.html

File: aio/tools/stackblitz-builder/builder.mjs (hunk 1 above)
Comment (alan-agius4) [reply aio/tools/stackblitz-builder/builder.mjs:185]: Looks like this has not been deleted.

**Review by alan-agius4 — APPROVED** [review]
LGTM thanks.

**Review by dylhunn — APPROVED** [review]

**Comment by dylhunn** [comment]
@alan-agius4 Please mark this for merge if you think it's good to go, thanks!

**Comment by dylhunn** [comment]
caretaker: @alan-agius4 and I are confident this is correct. We can merge it with current approvals.

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit 82c8b7ae9ece92cab891724a2e7314d18176abfe.

**Pull request closed without merging**